mods/voidmulti.sc2mod/base.sc2data/gamedata/modeldata.xml

```diff
@@ -352,6 +352,34 @@
             <Payload value="Repair_Bot_Beam_01"/>
         </Events>
     </CModel>
+    <CModel id="Observer">
+        <Radius value="0.438000"/>
+        <ScaleMax value="1.000000,1.000000,1.000000"/>
+        <ScaleMin value="1.000000,1.000000,1.000000"/>
+        <SelectionRadius value="0.515000"/>
+        <ShadowRadius value="0.515000"/>
+    </CModel>
+    <CModel id="ObserverPlacement">
+        <Radius value="0.438000"/>
+        <ScaleMax value="1.000000,1.000000,1.000000"/>
+        <ScaleMin value="1.000000,1.000000,1.000000"/>
+        <SelectionRadius value="0.515000"/>
+        <ShadowRadius value="0.515000"/>
+    </CModel>
+    <CModel id="PurificationNova">
+        <Radius value="2.291700"/>
+        <RadiusLoose value="2.291700"/>
+        <ScaleMax value="0.640000,0.640000,0.640000"/>
+        <ScaleMin value="0.640000,0.640000,0.640000"/>
+    </CModel>
+    <CModel id="PurificationNovaRangeIndicator">
+        <Radius value="2.291700"/>
+        <RadiusLoose value="2.291700"/>
+        <!--ScaleMax value="1.200000,1.200000,1.200000"/>
+        <ScaleMin value="1.200000,1.200000,1.200000"\-->
+        <ScaleMax value="0.687500,0.687500,0.687500"/>
+        <ScaleMin value="0.687500,0.687500,0.687500"/>
```

Comment:
Disruptor's Purifier Orb radius reduced from 1.5 to 1.35 (-10%)
- model's radius reduced from 2.5 to 2.2917 (-8.3%)
- model's visual radius (RadiusLoose) reduced from 2.5 to 2.2917 (-8.3%)
- model's scale reduced from 0.75 to 0.6875 (-8.3%)

It is unexpected that the range indicator has not been shrunk by 10% as well. However, it looks accurate in this video:
[comparison video](https://twitter.com/leisvanCT/status/1601697775716290560)